test-harness/gold-master-files/__path____no_auth_pets___method___POST___headers____Content-Type___application_json____body____id__1__petId__2__quantity__3__shipDate___2002-10-02T10_00_00-05_00___status___placed___complete__true__.json

```diff
@@ -12,14 +12,34 @@
     "statusText": "Unprocessable Entity",
     "headers": {
       "content-type": ["application/problem+json"],
-      "content-length": ["518"],
+      "content-length": ["614"],
       "connection": ["close"]
     },
     "body": {
       "type": "https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY",
       "title": "Invalid request body payload",
       "status": 422,
-      "detail": "Your request body is not valid: [{\"path\":[\"body\"],\"code\":\"required\",\"message\":\"should have required property 'name'\",\"severity\":0},{\"path\":[\"body\"],\"code\":\"required\",\"message\":\"should have required property 'photoUrls'\",\"severity\":0},{\"path\":[\"body\",\"status\"],\"code\":\"enum\",\"message\":\"should be equal to one of the allowed values\",\"severity\":0}]"
+      "detail": "Your request body is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.",
+      "validation": [
```

Comment:
The output looks so cool now, fantastic! 👍

Reply:
Agreed! Great job.